drivers/wifi/esp/esp.c

```diff
@@ -838,7 +840,7 @@ static int esp_init(struct device *dev)
 	data->iface_data.rx_rb_buf = &data->iface_rb_buf[0];
 	data->iface_data.rx_rb_buf_len = sizeof(data->iface_rb_buf);
 	ret = modem_iface_uart_init(&data->mctx.iface, &data->iface_data,
-				    DT_INST_0_ESPRESSIF_ESP_BUS_NAME);
+				    DT_INST_BUS_LABEL(0));
```

Comment:
Seem be having issues with this change: 

> /drivers/wifi/esp/esp.c: In function 'esp_init':
> ../../include/devicetree.h:209:40: error: 'DT_N_INST_0_espressif_esp_wifi_BUS_P_label' undeclared (first use in this function)

Reply:
Fixed here: https://github.com/zephyrproject-rtos/zephyr/pull/24004